haproxy19/tests/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+SKIPBUILD=${SKIPBUILD:-0}
+
+hab pkg install core/busybox-static
+hab pkg binlink core/busybox-static netstat
+
+hab pkg install --binlink core/bats
+source "${PLANDIR}/plan.sh"
```

Comment:
This results in `haproxy19/plan.sh: line 1: ../haproxy/plan.sh: No such file or directory` when run from the root of core-plans.  I don't know if there's a good solution here since we have a couple levels of `source` happening.